helm/reana-stable/values.yaml

```diff
@@ -0,0 +1,123 @@
+# Default values for reana.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# TODO Document this shuld be set for third party integrations and for
+# production deployments due to security issues.
+# reana_url: reana.cern.ch
```

Comment:
- reana_url twice
- `s/shuld/should/`
- do we care about adding licence headers to helm files?